### ISSUE TITLE
[Unnamed Game] 

### ISSUE THREAD

**Issue by SuperMarioROCKS** [thesupersonic16/HedgeModManager#247]
[HMM_Snapshot_00062021155115.txt](https://github.com/thesupersonic16/HedgeModManager/files/6619542/HMM_Snapshot_00062021155115.txt)
HMM Info:
```
    Version: 7.4-2
    Args: 
    StartDir: C:\Users\yokel\Downloads\yeah 2\
    Process Level: Administrator
    Game: Unnamed Game
```

Exception:
```
    Type: NullReferenceException
    Message: Object reference not set to an instance of an object.
    Source: HedgeModManager
    Function: System.Collections.Generic.List`1[HedgeModManager.SteamGame] SearchForGames(System.String)
    StackTrace: 
       at HedgeModManager.Steam.SearchForGames(String preference)
       at HedgeModManager.HedgeApp.Main(String[] args)
    InnerException: 
```

**Comment by SuperMarioROCKS** [comment]
literally did nothing than launch the application
getting a new version doesn't work
with or without admin it still fails

**Comment by SuperMarioROCKS** [comment]
and if you can tell by the title... its not caused by a game.

**Comment by Sajidur78** [comment]
Does it work if you run HMM from a game directory? Seems like its failing to search for a Steam game.

**Comment by zemurian-guy** [comment]
I actually tried that and got all this:

HMM Info:
```
    Version: 7.4-2
    Args: 
    StartDir: E:\_Steam_\steamapps\common\Sonic Generations\
    Process Level: User
    Game: Unnamed Game
```

Exception:
```
    Type: NullReferenceException
    Message: Object reference not set to an instance of an object.
    Source: HedgeModManager
    Function: System.Collections.Generic.List`1[HedgeModManager.SteamGame] SearchForGames(System.String)
    StackTrace: 
       at HedgeModManager.Steam.SearchForGames(String preference)
       at HedgeModManager.HedgeApp.Main(String[] args)
    InnerException: 
```

**Comment by Sajidur78** [comment]
Huh, this is an interesting issue, are there any `.dmp` files named HedgeModManager in `%localappdata%/crashdumps`? If so, could you upload the file?

**Comment by Sajidur78** [comment]
The issue's been fixed in 66b40ab41abc8446e91cb3b6e15062aa41718fda and release [7.4-3](https://github.com/thesupersonic16/HedgeModManager/releases/tag/7.4-3), Closing.
